File: db4oj/core/src/com/db4o/MetaField.java
/* Copyright (C) 2004   db4objects Inc.   http://www.db4o.com */

package com.db4o;

/**
 * Field MetaData to be stored to the database file.
 * Don't obfuscate.
 */
public class MetaField implements Internal{
	
	public String name;
	public MetaIndex index;
	
	public MetaField(){
	}
	
	public MetaField(String name){
		this.name = name;
	}
}

File: db4on/core/src/core/com/db4o/MetaField.cs
namespace com.db4o
{
	/// <summary>Field MetaData to be stored to the database file.</summary>
	/// <remarks>
	/// Field MetaData to be stored to the database file.
	/// Don't obfuscate.
	/// </remarks>
	/// <exclude></exclude>
	/// <persistent></persistent>
	public class MetaField : com.db4o.Internal
	{
		public string name;

		public com.db4o.MetaIndex index;

		public MetaField()
		{
		}

		public MetaField(string name)
		{
			this.name = name;
		}
	}
}
